src/command/info/info.c

```diff
@@ -155,7 +182,7 @@ stanzaStatus(const int code, bool backupLockHeld, Variant *stanzaInfo)
         FUNCTION_TEST_PARAM(VARIANT, stanzaInfo);
     FUNCTION_TEST_END();
 
-    ASSERT(code >= 0 && code <= 5);
+    ASSERT((code >= 0 && code <= 6) || code == 99);
```

Comment:
Normally I don't like hardcoded numbers but in this instance, I do. Two reasons:
1) The constant names are very long and this would span over a couple lines.
2) More importantly, it is much easier see with the actual numbers when they are consecutive or not and when I get an error that says 
`assertion '(code >= 0 && code <= 3) || code == 99' failed`
vs 
`assertion '(code >= INFO_STANZA_STATUS_CODE_OK && code <= INFO_STANZA_STATUS_CODE_MISSING_STANZA_DATA) || code == INFO_STANZA_STATUS_CODE_OTHER' failed`
With the first I can clearly see it's a code less than 0, more than 3 but not 99. In the second I have to go find out what the value of INFO_STANZA_STATUS_CODE_OK is, what the value of INFO_STANZA_STATUS_CODE_MISSING_STANZA_DATA is and what the value of INFO_STANZA_STATUS_CODE_OTHER is before I can even start figuring out what might  be the problem. 

As this is an assert and is compiled out in production, I want it to be easy and clear for coding and debugging so we can spend the least amount of time debugging any problems..

Reply:
Fair enough!